packages/atomic/src/components/ipx/atomic-ipx-refine-modal/atomic-ipx-refine-modal.tsx

```diff
@@ -0,0 +1,170 @@
+import {
+  BreadcrumbManager,
+  BreadcrumbManagerState,
+  buildBreadcrumbManager,
+  buildFacetManager,
+  buildQuerySummary,
+  FacetManager,
+  QuerySummary,
+  QuerySummaryState,
+} from '@coveo/headless';
+import {Component, h, State, Prop, Element, Watch, Host} from '@stencil/core';
+import {rectEquals} from '../../../utils/dom-utils';
+import {
+  BindStateToController,
+  InitializableComponent,
+  InitializeBindings,
+} from '../../../utils/initialization-utils';
+import {Button} from '../../common/button';
+import {Hidden} from '../../common/hidden';
+import {
+  getClonedFacetElements,
+  RefineModalCommon,
+} from '../../common/refine-modal/refine-modal-common';
+import {Bindings} from '../../search/atomic-search-interface/atomic-search-interface';
+
+/**
+ * @internal
```

Comment:
The `@internal` flag for the insight components is because we don't want them publicly documented: They are only meant to be used "internally" with the builder. We do not intend clients to (at least for now) use the insight atomic components directly.

We also use the flag when we are still building a new component/feature that is not yet ready to be released because it's still incomplete, or untested.

We should keep the flag for now (I think), but heads up for you: If the goal is to make this a public facing component, with accompanying documentation, then there is still work that needs to be done, and some coordination with the documentation team (@jpmarceau & co)

Reply:
Alright, I'll make sure to touch point with you guys so we can make a plan when/if we want to make the atomic components to be used directly by clients